h/templates/activity/search.html.jinja2

```diff
@@ -356,7 +356,46 @@
       </ol>
     {% else %}
       <div class="search-result-zero {%- if more_info %} search-result-hide-on-small-screens{% endif %}">
-        {{ zero_message }}
+        {% if zero_message == '__SHOW_GETTING_STARTED__' %}
+          <div>
+            <h2 class="search-result-zero__title">
+              {% trans %}How to get started{% endtrans %}
+            </h2>
+            <ol class="search-result-zero__list">
+              <li><p>
+                <a href="https://chrome.google.com/webstore/detail/hypothesis-web-pdf-annota/bjfhmglciegochdpefhhlphglcehbmek">
+                  {% trans %}Install our Chrome extension{% endtrans %}
+                </a>
+              </p></li>
+              <li><p>
```

Comment:
It's cos I want more whitespace between the list items! To look more like the design. I realise CSS can add whitespace between li's without them needing to have p's inside them but since the use of p's in li's is sanctioned by markdown (!) https://daringfireball.net/projects/markdown/syntax#list I figured this was acceptable :)